lib/capybara/node/actions.rb

```diff
@@ -45,13 +41,11 @@ def click_link(locator = nil, **options)
       #
       # Finds a button on the page and clicks it.
       # This can be any \<input> element of type submit, reset, image, button or it can be a
-      # \<button> element. All buttons can be found by their id, name, Capybara.test_id attribute, value, or title. \<button> elements can also be found
+      # \<button> element. All buttons can be found by their id, name, {Capybara.configure test_id} attribute, value, or title. \<button> elements can also be found
       # by their text content, and image \<input> elements by their alt attribute
       #
-      # @overload click_button([locator], **options)
-      #   @macro waiting_behavior
-      #   @param [String] locator      Which button to find
-      #   @param options     See {Capybara::Node::Finders#find_button}
+      # @param [String] locator      Which button to find
+      # @option (see Capybara::Node::Finders#find)
```

Comment:
Please leave as this was - including the waiting behavior, and linking to find_button rather than find